src/Header.jsx

```diff
@@ -226,8 +227,26 @@ const User = ({address, setAddress, open, setOpen, toggleOpen}) => {
           setLoggingIn(true);
           try {
             const {address, profile} = await ceramicApi.login();
-            // console.log('login', {address, profile});
             setAddress(address);
+            if (address) {
+              (async () => {
+                var tokenUri = [];
+                const res = await fetch(`${apiBackend}/tokenids?address=${address}`);

```

Comment:
This could be one request to get the uris, rather than O(n + 1) requests. Is there a reason for doing it this way?

Reply:
converted into single request. ✅